### PR TITLE
fix tlsproxy

### PR DESCRIPTION
**Pull request by mjudeikis**
```release-note
NONE
```

TLSProxy changes to enable usage for vmimage

### CODE REVIEW

File: pkg/entrypoint/tlsproxy/tlsproxy.go [hunk 1]
```diff
@@ -24,7 +22,7 @@ func (c *cmdConfig) validate() (errs []error) {
 	}
 	if c.password == "" && c.username != "" ||
 		c.password != "" && c.username == "" {
-		errs = append(errs, fmt.Errorf("if either USERNAME or PASSWORD environment variable is set, both must be set"))
+		errs = append(errs, fmt.Errorf("if either USERNAME or PASSWORD environment variable is set, both must be unset"))
```
Comment (Makdaam) [inline pkg/entrypoint/tlsproxy/tlsproxy.go]:
```suggestion
		errs = append(errs, fmt.Errorf("if either USERNAME or PASSWORD environment variable is unset, both must be unset"))
```

**Comment by ehashman** [comment]
e2e-create #1839 
upgrade-v5.2 seems to have failed on the postgres container not being ready yet, also looking flaky

/retest

**Pull request merged**